### DISCUSSION TITLE
Jellyfin/rclone/pcloud

### DISCUSSION THREAD

**Discussion by FarangFlix** [jellyfin/jellyfin#9988] [General]
I've been running Jellyfin with Google Workspace storage mounted with rclone for many years now.  I'm now wanting to change to pcloud.  I have no issues adding the Google drive folders to the library but for some reason, can't so this with the pclone mount.  I get the error "The path could not be found. Please ensure the path is valid and try again."
I spent a couple of days trying things I found on this topic and nothing works.  Mostly playing with different permission settings which mostly doesn't work with the pcloud mount.

Has anyone successfully got a Jellyfin/rclone/pcloud setup to work?

**Comment by humble-new-student-89** [comment]
@FarangFlix Thanks for this post.  I am facing same issue.  Could you please help if you were able to get it to work?
